tasks/update-aws-sdk-dependencies.rake

```diff
@@ -19,11 +19,12 @@ task 'update-aws-sdk-dependencies' do
   )
 
   # update the module autoloads
+  autoload_services = BuildTools::Services.select { |svc| svc.gem_name != 'aws-sdk-core' }
   BuildTools.replace_lines(
     filename: "#{$GEMS_DIR}/aws-sdk-resources/lib/aws-sdk-resources.rb",
     start: /# service gems/,
     stop: /# end service gems/,
-    new_lines: BuildTools::Services.map { |service|
+    new_lines: autoload_services.map { |service|
```

Comment:
Why is core ignored here?

Reply:
This is a little confusing - this is actually removing the bundled service gems from the autoload.  They have a gem name of aws-sdk-core (since they are bundled in it). I've added comments.